File: Luna.Models.Tasks.View/CardAttributes/TagView.cs
using Luna.Models.Tasks.Domain.CardAttributes;

namespace Luna.Models.Tasks.View.CardAttributes;

public class TagView
{
	public Guid Id { get; set; }

	public String Name { get; set; }

	public String Color { get; set; }

	public TagView(TagDomain tagDomain)
	{
		Id = tagDomain.Id;
		Name = tagDomain.Name;
		Color = tagDomain.Color;
	}
}

File: luna/src/models/card/tagView.ts
import {Guid} from "guid-typescript";

export default class TagView{
    constructor(id: Guid, name: string, color: string) {
        this.id = id;
        this.name = name;
        this.color = color;
    }

    public id: Guid;
    public name: string;
    public color: string;
}